app/src/main/java/com/android/unio/model/firestore/transform/Serialization.kt

```diff
@@ -0,0 +1,45 @@
+package com.android.unio.model.firestore.transform
+
+import com.android.unio.model.association.Association
+import com.android.unio.model.association.AssociationRepositoryFirestore
+import com.android.unio.model.event.Event
+import com.android.unio.model.event.EventRepositoryFirestore
+import com.android.unio.model.user.User
+import com.android.unio.model.user.UserRepositoryFirestore
+
+fun AssociationRepositoryFirestore.Companion.serialize(association: Association): Map<String, Any> {
+  return mapOf(
+      "uid" to association.uid,
+      "url" to association.url,
+      "acronym" to association.acronym,
+      "fullName" to association.fullName,
+      "description" to association.description,
+      "members" to association.members.list.value.map { it.uid })
```

Comment:
I know this has been said a lot already, but we'll have to extract these strings into our string value file, as we had discussed in last meeting. This will be required to be done for all the files anyway

Reply:
Depends if the file contains a `@Composable` function and thus a `Context`. If not, we'll have to wait for the assistant' answer to see if we can create a Resource Manager as explained in #56 : )

Reply:
You're right, for this specific case I don't think it is even useful as the purpose of this PR is to centralize these things. Adding them on a different file to only use them here is unnecessary